examples/client/tui/common.nim

```diff
@@ -209,6 +215,7 @@ const
     "deletewalletaccount": @["delete"],
     "getassets": @["assets"],
     "getcustomtokens": @["gettokens"],
+    "getprice": @["getprice"],
```

Comment:
No need to add to the aliased table if there's no alias for `getprice`.

Reply:
Done.